searx/answerers/__init__.py

```diff
@@ -1,13 +1,17 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
This AGPL comment was missing prior, so I added it to the files I worked on. Can be removed if not needed / acceptable.